src/main/resources/logback-spring.xml

```diff
@@ -37,17 +37,21 @@
             <appender-ref ref="consoleAppender"/>
         </logger>
 
-        <logger name="org.apache.catalina" additivity="false" level="INFO" >
-            <appender-ref ref="consoleAppender"/>
-        </logger>
+      <logger name="org.apache.catalina" additivity="false" level="INFO">
+        <appender-ref ref="consoleAppender"/>
+      </logger>
 
-        <logger name="org.flywaydb" additivity="false" level="INFO" >
-            <appender-ref ref="consoleAppender"/>
-        </logger>
+      <logger name="org.flywaydb" additivity="false" level="INFO">
+        <appender-ref ref="consoleAppender"/>
+      </logger>
 
-        <root level="INFO">
-            <appender-ref ref="consoleAppender"/>
-        </root>
+      <logger name="org.flywaydb.core.internal.sqlscript.DefaultSqlScriptExecutor" additivity="false" level="WARN">
```

Comment:
this stops the count of rows affected being output, which for community api happens a lot as loads of data is inserted